### PR TITLE
Create article with survey results

### PR DESCRIPTION
**Pull request by marionf**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Create article with survey results

### CODE REVIEW

File: news/_posts/2021-05-05-community-survey-results.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+![When do you contribute to the open-source project?](/assets/images/2021/05/when-do-you-contribute.png)
+
+### Sort the opportunities for the open-source project in order of importance 
+
+The 2 most important opportunities identified are the fix of the major prices, taxes, and cart rule issues.
+
+Then comes the update of the Symfony LTS version and the support of PHP 8.0
+
+![Sort the opportunities for the open-source project in order of importance](/assets/images/2021/05/opportunities-os-project.png)
```
Comment (prestascott) [inline news/_posts/2021-05-05-community-survey-results.md:46]: It's weird to not show the average # of each opportunity? 
E.g. #3.48 average for "Fix major prices & taxes issues"

File: news/_posts/2021-05-05-community-survey-results.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+And 21,4% of respondents do not contribute to the project.
+
+![How do you contribute to the open-source project?](/assets/images/2021/05/how-do-you-contribute.png)
+
+Remember that the best way to contribute to the open-source project is to open an issue when you are facing a problem and to fix it with a pull request if you have the technical skills. 
+If you are not a developer, you can consider hiring a specialist to help you. If you manage to get the problem fixed, then contribute the fix back to the project as a Pull Request. Remember that the more people contribute, the better PrestaShop becomes for everyone :muscle:
+
+### When do you contribute to the open-source project?
+
+54,5% of the respondents contribute during their working time, while 45,5% contribute in their free time.
```
Comment (prestascott) [inline news/_posts/2021-05-05-community-survey-results.md]: 54.5% of the respondents contribute during their working time, while 45.5% contribute in their free time.

File: news/_posts/2021-05-05-community-survey-results.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Results of community survey – April 2021"
+subtitle: "Thank you for your feedback about the open-source project and the contributions"
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
subtitle: "Thank you for your feedback about the open source project and the contributions"
```
Open source has no `-`
but e-commerce has `-`
😉

File: news/_posts/2021-05-05-community-survey-results.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+date: 2021-05-05 12:00:00
+authors: [MarionFrancois]
+icon: icon-world
+image: /assets/images/2021/05/build-survey-banner.png
+---
+
+The core product team carried out a survey in April, oriented around the open-source project and for the community. The goal of this survey was to better know and understand the pains and the wishes of the community about the open-source project.
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
The core product team carried out a survey in April, concerning the open source project and for the community. The goal of this survey was to better know and understand the pains and the wishes of the community about the open source project.
```

File: news/_posts/2021-05-05-community-survey-results.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+![Build survey banner](/assets/images/2021/05/build-survey-banner.png)
+
+## :two_hearts: Special thanks to the participants
+
+We received 98 responses, which makes a completion rate of 28,6%, and wanted to thank everyone who participated in this survey. Your opinions are very important for us and we care a lot about your satisfaction!
+
+## Results of the community survey of April 2021
+
+### How do you contribute to the open-source project?
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
### How do you contribute to the open source project?
```

File: news/_posts/2021-05-05-community-survey-results.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+authors: [MarionFrancois]
+icon: icon-world
+image: /assets/images/2021/05/build-survey-banner.png
+---
+
+The core product team carried out a survey in April, oriented around the open-source project and for the community. The goal of this survey was to better know and understand the pains and the wishes of the community about the open-source project.
+
+![Build survey banner](/assets/images/2021/05/build-survey-banner.png)
+
+## :two_hearts: Special thanks to the participants
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
## :two_hearts: Special thanks to the respondents
```

File: news/_posts/2021-05-05-community-survey-results.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+We received 98 responses, which makes a completion rate of 28,6%, and wanted to thank everyone who participated in this survey. Your opinions are very important for us and we care a lot about your satisfaction!
+
+## Results of the community survey of April 2021
+
+### How do you contribute to the open-source project?
+
+The answers to the first question show that the majority of contributors report bugs and suggest improvements or new features for the open-source project.
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
The answers to the first question show that the majority of contributors report bugs and suggest improvements or new features for the open source project. 
```

File: news/_posts/2021-05-05-community-survey-results.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+The code contribution with pull requests comes in 3rd position with 35,7%.
+
+And 21,4% of respondents do not contribute to the project.
+
+![How do you contribute to the open-source project?](/assets/images/2021/05/how-do-you-contribute.png)
+
+Remember that the best way to contribute to the open-source project is to open an issue when you are facing a problem and to fix it with a pull request if you have the technical skills. 
+If you are not a developer, you can consider hiring a specialist to help you. If you manage to get the problem fixed, then contribute the fix back to the project as a Pull Request. Remember that the more people contribute, the better PrestaShop becomes for everyone :muscle:
+
+### When do you contribute to the open-source project?
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
### When do you contribute to the open source project?
```

File: news/_posts/2021-05-05-community-survey-results.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+54.5% of the respondents contribute during their working time, while 45.5% contribute in their free time.
+
+![When do you contribute to the open-source project?](/assets/images/2021/05/when-do-you-contribute.png)
+
+### Sort the opportunities for the open-source project in order of importance 
+
+The 2 most important opportunities identified are the fix of the major prices, taxes, and cart rule issues.
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
The two most important needs identified are the major issues related to prices, taxes, and cart rules.
```

File: news/_posts/2021-05-05-community-survey-results.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+Remember that the best way to contribute to the open-source project is to open an issue when you are facing a problem and to fix it with a pull request if you have the technical skills. 
+If you are not a developer, you can consider hiring a specialist to help you. If you manage to get the problem fixed, then contribute the fix back to the project as a Pull Request. Remember that the more people contribute, the better PrestaShop becomes for everyone :muscle:
+
+### When do you contribute to the open-source project?
+
+54.5% of the respondents contribute during their working time, while 45.5% contribute in their free time.
+
+![When do you contribute to the open-source project?](/assets/images/2021/05/when-do-you-contribute.png)
+
+### Sort the opportunities for the open-source project in order of importance
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
### Sort the opportunities for the open source project in order of importance 
```

File: news/_posts/2021-05-05-community-survey-results.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+54.5% of the respondents contribute during their working time, while 45.5% contribute in their free time.
+
+![When do you contribute to the open-source project?](/assets/images/2021/05/when-do-you-contribute.png)
+
+### Sort the opportunities for the open-source project in order of importance 
+
+The 2 most important opportunities identified are the fix of the major prices, taxes, and cart rule issues.
+
+Then comes the update of the Symfony LTS version and the support of PHP 8.0
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
Then comes the update of the [Symfony](https://symfony.com/) [LTS](https://en.wikipedia.org/wiki/Long-term_support) version and the support of PHP 8
```

No need to mention PHP 8.0 ; only PHP8 : then will come PHP 8.1 ...

File: news/_posts/2021-05-05-community-survey-results.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+![Sort the opportunities for the open-source project in order of importance](/assets/images/2021/05/opportunities-os-project.png)
+
+### Sort the technical subjects in order of importance
+
+The first subject that goes back to this question is the fix of major bugs.
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
The first topic that emerged from this question is the fix of major known bugs.
```

File: news/_posts/2021-05-05-community-survey-results.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+Then comes the simplification of integrating development in the back-office and extending PrestaShop.
```
Comment (matks) [inline news/_posts/2021-05-05-community-survey-results.md]:
```suggestion
Then comes the easibility of customizing the back-office and extending PrestaShop.
```

File: news/_posts/2021-05-05-community-survey-results.md (hunk 1 above)
Comment (marionf) [reply news/_posts/2021-05-05-community-survey-results.md:46]: Yes, I just update the images :)

File: news/_posts/2021-05-05-community-survey-results.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+layout: post
+title: "Results of community survey – April 2021"
+subtitle: "Thank you for your feedback about the open source project and the contributions"
+date: 2021-05-05 12:00:00
+authors: [MarionFrancois]
+icon: icon-world
+image: /assets/images/2021/05/build-survey-banner.png
+---
+
+The core product team carried out a survey in April, concerning the open source project and for the community. The goal of this survey was to better know and understand the pains and the wishes of the community about the open source project.
```
Comment (ldeprestashop) [inline news/_posts/2021-05-05-community-survey-results.md]: "and for" 
the open source for the community ?

Comment (marionf) [reply news/_posts/2021-05-05-community-survey-results.md]: Fixed, thank you

**Pull request merged**